README.md

```diff
@@ -806,6 +806,13 @@ nbodykit is an open source project written in Python that provides a set of stat
 
 [nbodykit Website](https://nbodykit.readthedocs.io/en/latest/index.html)
 
+
+### pNbody
+
+pNbody is a parallelized python module toolbox designed to manipulate and display interactively very large N-body systems. Its oriented object approach allows the user to perform complicate manipulation with only very few commands. Some additional packages like [glups](https://gitlab.com/revaz/glups) (Python/OpenGL GUI designed to display Nbody models), [Gtools](https://gitlab.com/revaz/Gtools) (a suite of tools and scripts designed to reduce n-body systems representing galaxies) and [Mtools](https://gitlab.com/revaz/Mtools) (routines to adapt images for movies).
```

Comment:
```suggestion
pNbody is a parallelized python module toolbox designed to manipulate and display interactively very large N-body systems. Its oriented object approach allows the user to perform complicated manipulations with only very few commands. Some additional packages like [glups](https://gitlab.com/revaz/glups) (Python/OpenGL GUI designed to display N-body models), [Gtools](https://gitlab.com/revaz/Gtools) (a suite of tools and scripts designed to reduce N-body systems representing galaxies) and [Mtools](https://gitlab.com/revaz/Mtools) (routines to adapt images for movies).
```

Reply:
> Some additional packages like [..]

I don't quite understand what this means (it is not a complete sentence). Are some additional packages available? Are they plugins for pNbody? Are there other packages/plugins?

Reply:
Sorry, my bad. It's been a long day. Better?

Reply:
Perfect, thanks for clarifying.